kube/services/fence/fence-deploy.yaml

```diff
@@ -90,6 +93,12 @@ spec:
           value: /var/www/fence
         - name: GEN3_DEBUG
           GEN3_DEBUG_FLAG|-value: "False"-|
+        - name: PRIVACY_POLICY_URL
```

Comment:
there are other fence deployment yamls you may need to update with this information. you can just grep `name: fence` to see where fence containers are used, noteably, you probably want to update the `fence-canary-deploy`. Unsure if `fenceshib-deploy` really needs this.. there are also a bunch of other jobs that include fence but most of those just call out to fence-create. So long as fence behaves when this config is not there then that should be fine

Reply:
Yeah, probably good to add to the canary (and maybe fenceshib?). That's correct, it isn't necessary for pods running fence for jobs like usersync

Reply:
think I will leave out fenceshib for now since this feature won't be available on it until backport/update anyways